properties.schema

```diff
@@ -19,20 +19,12 @@
                 "_isEnabled": {
                   "type":"boolean",
                   "required":true,
+                  "default": true,
                   "title":"Is Enabled",
                   "inputType": {"type": "Boolean", "options": [true, false]},
                   "validators": [],
                   "help": "If set to 'true', this course will be tracked by TinCan/xAPI"
                 },
-                "_activityID": {
```

Comment:
I think we still need this in the schema, since users will be required to add it so the manifest.xml file has the right value when we build the course. If it's required do we need to provide a default? Can we default to a calculated value?